src/renderer/qss/js/qssServiceButtons.js

```diff
@@ -225,7 +225,9 @@
                 funcName: "gpii.windows.openUrl",
                 args: [
                     "{that}.model.item.schema.url", // using the url from the custom button's schema
-                    "{gpii.qss}.options.siteConfig.alwaysUseChrome", // Override the OS default browser.
+                    true, // Override the OS default browser to always use Chrome instead
+                    // if you want to use the value from site config this is:
```

Comment:
Please add a link to https://issues.gpii.net/browse/GPII-4422 where this reasoning is explained

Reply:
Done, added to the comment just below the variable to explain the reasoning.